buildmodel/buildassets/buildassets_test.go

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package buildassets
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"reflect"
+	"strings"
+	"testing"
+
+	"github.com/microsoft/go-infra/buildmodel/dockerversions"
+)
+
+func TestBuildResultsDirectoryInfo_CreateSummary(t *testing.T) {
```

Comment:
FYI: I created a new test func with a simple baseline to help verify the changes I made to `CreateSummary` when removing `panic`s. It's not meant to be extensive, I just didn't want to continue having absolutely nothing for this code path. 😄